packages/website/components/account/storageManager/storageManager.js

```diff
@@ -34,39 +28,23 @@ const mailTo = `mailto:${emailContent.mail}?subject=${emailContent.subject}&body
  * @returns
  */
 const StorageManager = ({ className = '', content }) => {
-  const storageTier = StorageTiers.TIER_1; // No tier available?
   const {
     storageData: { data, isLoading },
   } = useUser();
   const uploaded = useMemo(() => data?.usedStorage?.uploaded || 0, [data]);
   const pinned = useMemo(() => data?.usedStorage?.pinned || 0, [data]);
+  const limit = useMemo(() => data?.storageLimitBytes || defaultStorageLimit, [data]);
   const [componentInViewport, setComponentInViewport] = useState(false);
   const storageManagerRef = useRef(/** @type {HTMLDivElement | null} */ (null));
 
   const { maxSpaceLabel, unlockLabel, percentUploaded, percentPinned } = useMemo(
-    () =>
-      // Storage information by tier
-      ({
-        [StorageTiers.TIER_1]: {
-          maxSpaceLabel: content.tiers[0].max_space_label,
-          unlockLabel: content.tiers[0].unlock_label,
-          percentUploaded: (uploaded / tebibyte) * 100,
-          percentPinned: (pinned / tebibyte) * 100,
-        },
-        [StorageTiers.TIER_2]: {
-          maxSpaceLabel: content.tiers[1].max_space_label,
-          unlockLabel: content.tiers[1].unlock_label,
-          percentUploaded: (uploaded / (tebibyte * 10)) * 100,
-          percentPinned: (pinned / (tebibyte * 10)) * 100,
-        },
-        [StorageTiers.TIER_3]: {
-          maxSpaceLabel: `${Math.floor(uploaded + pinned / (tebibyte * 10) + 1) + content.tiers[2].max_space_label}`,
-          // every increment of 10 changes the amount of space used
-          percentUploaded: ((uploaded % (tebibyte * 10)) / (tebibyte * 10)) * 100,
-          percentPinned: ((pinned % (tebibyte * 10)) / (tebibyte * 10)) * 100,
-        },
-      }[storageTier]),
-    [storageTier, uploaded, pinned, content.tiers]
+    () => ({
+      maxSpaceLabel: `${limit / tebibyte} ${content.max_space_tib_label}`,
```

Comment:
Should we `Math.floor(limit / tebibyte)` in case we end up with a long floating point number?

Reply:
Good point. Done.